reviews/tests.py

```diff
@@ -465,6 +460,36 @@ def setUp(self):
             ),
         )
 
+    def test_auto_review_pre_assessment_short_route(self):
+        # pre-assessment is supposed to go short route
+        reasons = auto_review(self.proposal)
+        self.assertEqual(len(reasons), 0)
+
+    def test_auto_review(self):
+        # tests for multiple cases pre-assessment should not be able to handle
+        pass
+
+    def test_auto_review_minors_to_longroute(self):
+        # pre-assessment has no age group
+        pass
+
+    def test_auto_review_adults_to_short_route(self):
+        # pre-assessment has no age group
+        pass
+
+    def test_auto_review_session_time(self):
+        # pre-assessment actually passes session time if you let this test trough,
+        # most likely because of self.study.has_sessions = True
+        # which can not happen in a real scenario for pre-assessment
```

Comment:
I like that you added comments to all these pass statements! However, please keep the future programmer in mind when writing comments. This specific comment is likely to cause more confusion than enlightenment.